baselines/aws/aws_mods/variables.tf

```diff
@@ -1,73 +1,88 @@
 variable "mod_list" {
-    description = <<DESC
-  Map of the list of approved regions controls.
-
-  Possible values for the key of this map is found in locals.tf defined in the local variable policy_map.
-  For example `aws-acm-certificate : "tmod:@turbot/aws-acm#/policy/types/certificateApproved"` has key `aws-acm-certificate`.
-
-  The value of the map is one of these possible values:
-    - "Skip"
-    - "Check: Approved"
-    - "Enforce: Delete unapproved if new"
-
-  Check demo.tfvars for an example of how to set this value.
-
-  NOTE: Default behaviour is to approve all services which means expecting all mods to be installed
-  DESC
-  type = list(string)
+  type        = list(string)
+  description = "The list of AWS Mods to install."
   default = [
```

Comment:
the default.tfvars has the shorter list required to get started with CIS atleast.. The intention of having the default in here is to show case the complete list of possibilities.. 

If we do not want it here, then I can update the README accordingly and redirect to [Guardrails Hub](https://hub.guardrails.turbot.com/) for complete list of Mods/services supported?

Reply:
I like the idea of using only default.tfvars with complete list and commenting out the ones not on by default.. but we will have to update the baselines as in when a new mod/policy is updated, whereas Guardrails Hub does that automatically. For an end-user just commenting/uncommenting the default.tfvars would be much easier rather than going back and forth between Guardrails Hub and guardrails-samples!